.tool-versions

```diff
@@ -1,3 +1,3 @@
-elixir 1.9.1
+elixir 1.9.1-otp-22
```

Comment:
Asking because I don't know - Do we still need to specify erlang below?

Reply:
According to <https://elixir-lang.org/install.html#precompiled-package>, the first step is to install erlang. Therefore, we want to specify our erlang version with asdf. The erlang postfix designates which erlang version was used to compile the precompiled elixir version.

I also found <https://github.com/asdf-vm/asdf-elixir#elixir-precompiled-versions> which states that you still need to specify your erlang version as well.